spec/api-remote-spec.js

```diff
@@ -0,0 +1,11 @@
+const assert = require('assert')
+const electron = require('electron')
+
+describe('remote module', function () {
+  describe('remoteMemberFunction.toString()', function () {
```

Comment:
Would you mind moving this to `api-ipc-spec.js` near the other `remote` specs? I know the name isn't super clear but that is where the other remote function specs are.